module/VuFind/sql/pgsql.sql

```diff
@@ -47,7 +47,7 @@ DROP TABLE IF EXISTS "resource_tags";
 
 CREATE TABLE resource_tags (
 id SERIAL,
-resource_id int NOT NULL DEFAULT '0',
+resource_id int DEFAULT NULL,
```

Comment:
We should also add a PostgreSQL migration for this change; I've added a TODO list to the PR so we don't forget to do this. I'm perfectly happy to hold off on that detail until closer to the end of the project, so we don't have to revise it if we decide to make any further changes in the meantime.